src/scss/components/menu/_menu.scss

```diff
@@ -2,127 +2,127 @@
+  &[data-position='centered'] {
+    nav {
+        background: $ui-menu-second-background-color;
 
-            ul {
-                @include outer-container($small-breakpoint);
+      ul {
+          @include outer-container($small-breakpoint);
 
-                @include media($medium-up) {
-                    @include outer-container($medium-breakpoint);
-                }
+          @include media($medium-up) {
+              @include outer-container($medium-breakpoint);
+          }
 
-                @include media($large-up) {
-                    @include outer-container($large-breakpoint);
-                }
-            }
-        }
+          @include media($large-up) {
+            @include outer-container($large-breakpoint);
+          }
+      }
     }
+  }
 
-    &[data-position='right'] {
-        nav {
-            float: left;
+  &[data-position='right'] {
+    nav {
+      float: left;
+      width: 100%;
+      @include outer-container(100%);
+      background: $ui-menu-second-background-color;
+
+      ul {
+        float: right;
+        background: $ui-menu-second-background-color;
+
+        li {
+          &:hover > ul {
+            display: inline-block;
             width: 100%;
-            @include outer-container(100%);
-            background: $ui-menu-second-background-color;
+          }
 
-            ul {
-                float: right;
-                background: $ui-menu-second-background-color;
-
-                li {
-                    &:hover > ul {
-                          display: inline-block;
-                          width: 100%;
-                    }
-
-                    // level 2
-                    ul {
-                        li {
-                            // level 3 - 4
-                            ul {
-                                position: relative;
-                                left: -100%;
-                                display: none;
-                                top: -$ui-menu-height;
-                            }
-                        }
-                    }
-                }
+        // level 2
+          ul {
+            li {
+              // level 3 - 4
+              ul {
+                position: relative;
+                left: -100%;
+                display: none;
+                top: -$ui-menu-height;
+              }
             }
+          }
         }
+      }
     }
+  }
+
+  nav {
+    width: 100%;
+
+    ul {
+      height: $ui-menu-height;
+      position: relative;
+      @include outer-container(100%);
+      background: $ui-menu-second-background-color;
+
+      li {
+        float: left;
+        text-align: center;
+        position: relative;
+
+        a,
+        div {
+          @include pad($ui-menu-link-padding);
+          width: 100%;
+          line-height: $ui-menu-link-line-height;
+          display: block;
+          cursor: pointer;
+          color: $ui-menu-link-color;
+
+          &:hover, &:link, &:focus {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 5<br>Each selector in a comma sequence should be on its own line